### PR TITLE
zh-translation:zh/docs/ops/best-practices/security/index.md #5990

### PR DESCRIPTION
**Pull request by mossinfan**

### CODE REVIEW

File: content/zh/docs/ops/best-practices/security/index.md [hunk 1]
```diff
@@ -1,43 +1,30 @@
...
+description: 使用 Istio 保护应用程序的最佳做法。
 force_inline_toc: true
 weight: 30
 ---
 
-This section provides some deployment guidelines to help keep a service mesh secure.
+本章节提供了一些部署准则，以帮助确保服务网格的安全。
 
-## Use namespaces for isolation
+## 使用命名空间进行隔离
```
Comment (ilylia) [inline content/zh/docs/ops/best-practices/security/index.md]: 缺少锚点

File: content/zh/docs/ops/best-practices/security/index.md [hunk 2]
```diff
@@ -1,43 +1,30 @@
...
 
-If there are multiple service operators (a.k.a. [SREs](https://en.wikipedia.org/wiki/Site_reliability_engineering))
-deploying different services in a medium- or large-size cluster, we recommend creating a separate
-[Kubernetes namespace](https://kubernetes.io/docs/tasks/administer-cluster/namespaces-walkthrough/) for each SRE team to isolate their access.
-For example, you can create a `team1-ns` namespace for `team1`, and `team2-ns` namespace for `team2`, such
-that both teams cannot access each other's services.
+如果有多个服务运营商（又称 [SREs](https://en.wikipedia.org/wiki/Site_reliability_engineering))在一个中型或者大型的集群中部署不同的服务，我们建议为每一个 SRE 团队创建一个单独的 [Kubernetes namespace](https://kubernetes.io/docs/tasks/administer-cluster/namespaces-walkthrough/) 以隔离其访问权限。
```
Comment (ilylia) [inline content/zh/docs/ops/best-practices/security/index.md]:
```suggestion
如果有多个服务运维人员（又称 [SREs](https://en.wikipedia.org/wiki/Site_reliability_engineering))在一个中型或者大型的集群中部署不同的服务，我们建议为每一个 SRE 团队创建一个单独的 [Kubernetes 命名空间](https://kubernetes.io/docs/tasks/administer-cluster/namespaces-walkthrough/) 以隔离其访问。
```

File: content/zh/docs/ops/best-practices/security/index.md [hunk 3]
```diff
@@ -1,43 +1,30 @@
...
+例如，你可以为 `team1` 创建一个 `team1-ns` 的命名空间，并为 `team2` 创建 `team2-ns` 命名空间， 这样，两个团队就无法访问彼此的服务。
 
 {{< warning >}}
-If Citadel is compromised, all its managed keys and certificates in the cluster may be exposed.
-We **strongly** recommend running Citadel in a dedicated namespace (for example, `istio-citadel-ns`), to restrict access to
-the cluster to only administrators.
+如果 Citadel 受到威胁，则集群中所有被他托管的密钥和证书都可能会被公开。
```
Comment (ilylia) [inline content/zh/docs/ops/best-practices/security/index.md]:
```suggestion
如果 Citadel 受到威胁，则集群中所有被其托管的密钥和证书都可能会被公开。
```

**Comment by mossinfan** [comment]
Hi ilylia:
thank you for your kindly comments.
I have fixed as your suggest.
Would you please help to review again, thanks.

**Comment by ilylia** [comment]
请参考 https://github.com/servicemesher/istio-official-translation#step3%E7%BF%BB%E8%AF%91 锚点小节的说明添加英文锚点

File: content/zh/docs/ops/best-practices/security/index.md [hunk 4]
```diff
@@ -1,43 +1,30 @@
...
-that both teams cannot access each other's services.
+如果有多个服务运维人员（又称 [SREs](https://en.wikipedia.org/wiki/Site_reliability_engineering))在一个中型或者大型的集群中部署不同的服务，我们建议为每一个 SRE 团队创建一个单独的 [Kubernetes 命名空间](https://kubernetes.io/docs/tasks/administer-cluster/namespaces-walkthrough/) 以隔离其访问。
+例如，你可以为 `team1` 创建一个 `team1-ns` 的命名空间，并为 `team2` 创建 `team2-ns` 命名空间， 这样，两个团队就无法访问彼此的服务。
 
 {{< warning >}}
-If Citadel is compromised, all its managed keys and certificates in the cluster may be exposed.
-We **strongly** recommend running Citadel in a dedicated namespace (for example, `istio-citadel-ns`), to restrict access to
-the cluster to only administrators.
+如果 Citadel 受到威胁，则集群中所有被其托管的密钥和证书都可能会被公开。
+我们 **强烈** 建议你在一个专门的命名空间（例如，“istio-citadel-ns”）中运行 Citadel，限制该集群只有管理员才可以访问。
```
Comment (ilylia) [inline content/zh/docs/ops/best-practices/security/index.md]:
```suggestion
我们 **强烈** 建议你在一个专门的命名空间（例如，`istio-citadel-ns`）中运行 Citadel，限制该集群只有管理员才可以访问。
```

**Comment by yuxiaobo96** [comment]
@ilylia 
This translation document, and two other documents (#6132, #6112) translated by mossinfan, look at the document translation records. Did you help synchronize and translate?

**Comment by ilylia** [comment]
@yuxiaobo96 some other translators had translated these three docs. I think you can just close them.

**Comment by yuxiaobo96** [comment]
Ok, I will close this pr.

**Pull request closed without merging**